v6/v6_interface.ml

```diff
@@ -20,17 +20,22 @@ let uri () = "file:" ^ !default_path
 
 type debug_info = string
 
+type edition_info = {
+    edition: string;
+    licence_restrictions: (string * string) list;
```

Comment:
Actually, it looks like the one below is the one that contains more than just restriction. But the same argument applies.